cypress/integration/malicious-proposal-validation.spec.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference path="../support/index.d.ts" />
+
+import { EPOCH_LENGTH } from '../support/common';
+
+describe('Malicious proposal validation', () => {
+  it('shows a warning if the original EVM script and the decoded EVM script does not match', () => {
+    cy.increaseTimeAndRelogin(EPOCH_LENGTH + 60 * 60); // skip the genesis epoch (add 1 hour just to be sure)
+    /*
+      Original EVM script: transfer(address,uint256)
+      Decoded EVM script: transfer(address,int32)
+     */
```

Comment:
It would be nice to have an additional test where the decoded EVM script specifies an additional function call as an attack (for example by specifying the same function call twice). This should do (I copy-pasted everything past `0x00000001`, which should have the proposal transfer double the amount of tokens it seems to):
```
0x00000001556ecbb0311d350491ba0ec7e019c354d7723ce0000000e4b61d27f6000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000044a9059cbb000000000000000000000000cb943e4fb0bcf7ec3c2e6d263c275b27f07701c600000000000000000000000000000000000000000000000000000019edcabff000000000000000000000000000000000000000000000000000000000556ecbb0311d350491ba0ec7e019c354d7723ce0000000e4b61d27f6000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000044a9059cbb000000000000000000000000cb943e4fb0bcf7ec3c2e6d263c275b27f07701c600000000000000000000000000000000000000000000000000000019edcabff000000000000000000000000000000000000000000000000000000000
```

Reply:
Ok cool will do 👍